_dev/src/components/commons/popin-configured.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <b-card
+    class="mb-2 shadow-sm border-0"
+    style="max-width: 40rem"
+  >
+    <b-card-body align="center">
+      <h3 class="font-weight-600">
+        {{ $t("configuredState.title") }}
+      </h3>
+      <img
```

Comment:
You might want to use the element `b-img` with the attribute `fluid` to be sure that the image won't overflow the popin on a small device.

`  <b-img src="@/assets/images/configured.png" fluid alt="" />`